conjure-java-jersey-server/src/main/java/com/palantir/conjure/java/server/jersey/ConjureJerseyFeature.java

```diff
@@ -35,6 +35,7 @@ public boolean configure(FeatureContext context) {
         context.register(new IllegalArgumentExceptionMapper());
         context.register(new NoContentExceptionMapper());
         context.register(new RuntimeExceptionMapper());
+        context.register(new WebApplicationExceptionMapper());
```

Comment:
Can we also add the javadoc explanation for why we need this to the top of the WebApplicationExceptionMapper class?

Reply:
Sure. I'd also like to remove the log.error from this mapper, since people are gonna be really confused about it